### PR TITLE
create a reusable timeline component

### LINKED ISSUE #62: Create a reusable Timeline component
Using timelines to describe procedures and milestones is required almost in every website in recent times. We would also like to build a reusable timeline component that would ease up the process of generating websites quickly.

I am working on this and expected to open a PR soon for the same.

### PR DESCRIPTION
**Pull request by Grumpyyash**
Created a highly reusable timeline component that can be customized using props and styles.
This PR resolves #62 
In Raw form, it looks like this - 

timeline-component

Headers and text and style can be customized using props and data to be passed as - 

code

### CODE REVIEW

**Comment by Grumpyyash** [comment]
Could you please review this @wdevon99 @charithccmc ? Any suggestions by the community would be highly appreciated.

File: src/components/TimelineComponent/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		return (
+      <div className="timeline-container">
+            {header ? <div className="header-component"><h1 className="title">{header}</h1></div> : null}
+            {data.map((data, idx) => (
+                <TimelineItem data={data} key={idx} />
+            ))}
+      </div>
+		);
+}
+
```
Comment (wdevon99) [inline src/components/TimelineComponent/index.js:39]: Fix indentation please :)

**Review by wdevon99 — APPROVED** [review]
LGTM, please check comments

**Comment by Grumpyyash** [comment]
Will improve in a fresh PR :)

**Pull request closed without merging**